### ISSUE TITLE
Only AP > 0 for one class out of 36

### ISSUE THREAD

**Issue by ozzyou** [johschmidt42/PyTorch-Object-Detection-Faster-RCNN-Tutorial#4]
Dear John,
Thanks for the very clear and nice tutorial. I am currently trying to implement Faster-RCNN on ActionGenome using your implementation. This dataset has 36 classes. I managed to get the model to train by setting the right parameters in training_script.py, however I'm only getting an AP > 0 for just one class, the "person" class (around 0.55). The dataset is a bit unbalanced towards that class, but not severely. I've checked the ground truth annotations, how they look before being passed to the model and everything seems to be fine. Since your tutorial was just for one class, I was wondering if you ever experienced similar issues. If yes, how did you solve them? Thanks!

**Comment by ozzyou** [comment]
Unfortunately I could not visualize in napari, but I used matplotlib on the generated anchors. The first image is what I used for training: anchor_size=(15, 30, 60, 120, 240)/aspect_ratios=(0.5, 1.0, 2.0), while the second picture is with anchor_size=(60, 120, 240)/aspect_ratios=(0.5, 1.0, 2.0). It gives the same 0 AP. Seems like something is off with the anchor points. The images are not scaled or normalized.
![image](https://user-images.githubusercontent.com/27645665/137312478-df460fea-9ee7-4cab-9fb2-aae209e7ce76.png)
![image](https://user-images.githubusercontent.com/27645665/137312493-e2f135c0-662e-48db-a6ab-1dd70e430ded.png)
